README.md

```diff
@@ -23,7 +23,7 @@ GET /search
 
 | Name | Type | Description | Example |
 | ------- | ------- | ------- | ------- |
-| **matches** | *string* | keyword(s) to search for | `"java"`
+| **matches** | *string* | keyword(s) to search for | `"projectriff"`
```

Comment:
We should get rid of the `projectriff` references because it's been EOL'ed. If the "java" doesn't match up somewhere else, let's change those places instead.